trcli/api/project_based_client.py

```
@@ -214,19 +213,35 @@ def create_or_update_test_run(self) -> Tuple[int, str]:
+                start_date=self.environment.run_start_date,
+                end_date=self.environment.run_end_date,
                 plan_id=self.environment.plan_id,
                 config_ids=self.environment.config_ids,
                 assigned_to_id=self.environment.run_assigned_to_id,
                 include_all=bool(self.environment.run_include_all),
                 refs=self.environment.run_refs,
+                case_ids=self.environment.run_case_ids,
             )
             run_id = added_run
         else:
             self.environment.log(f"Updating test run. ", new_line=False)
```

Ruff (0.14.3)

f-string without any placeholders

Remove extraneous `f` prefix

(F541)